docs/product/components/avatars.html

```diff
@@ -154,39 +154,45 @@
 {% highlight html %}
 <a href="…" class="s-link s-link__muted">
     <div class="s-avatar">
-        <div class="s-avatar--letter">H</div>
+        <div class="s-avatar--letter" aria-hidden="true">H</div>
```

Comment:
I'm guessing these are hidden from screenreaders because they're "images" and therefore should not be read aloud?

Reply:
Yep, these letter avatars should include the team name wrapped in a node including `.v-visible-sr`.

Reply:
Thanks for taking a look btw!